README.md

```diff
@@ -45,6 +45,12 @@ Clone or fork the Heptio Ark repo:
 git clone git@github.com:heptio/ark.git
 ```
 
+Checkout the appropriate tag for the Ark version you wish to use (e.g. replacing `<VERSION TAG>` with "v0.3.3"):
+```
+git checkout <VERSION TAG>
+```
+You should `git checkout master` only if you're planning on [building the Ark image from scratch][7].
```

Comment:
If you are building from scratch and you want to test your custom image locally, it's a bit annoying, because:

1. You deploy our deployment
1. Because it specifies `gcr.io/heptio-images/ark:latest`, it defaults the image pull policy to `Always`
1. The kubelet pulls from gcr.io because of the above image pull policy, losing your changes
1. You rebuild your container again
1. You have to edit the deployment to change the pull policy to IfNotPresent
1. You redeploy the deployment (scale down & back up)